codespell_lib/data/dictionary.txt

```diff
@@ -51229,6 +51229,8 @@ spoace->space
 spoaced->spaced
 spoaces->spaces
 spoacing->spacing
+spoffed->spoofed
```

Comment:
```suggestion
spoff->spoof, spiff, spaff, scoff,
spoffed->spoofed, spaffed, scoffed,
```

Reply:
Thanks a lot for the suggestion. :+1:

Unfortunately i wasn't able to apply the suggestion directly in GitHub because it always showed me a "Diff changed, please refresh" and showed that recurring after each refresh (probably some kind of bug in GitHub) so i had applied the suggestions now in / via 98693210b75bceb24a0dff16ad220e551318be81